pkg/apis/kops/cluster.go

```diff
@@ -319,6 +319,8 @@ type ClusterSubnetSpec struct {
 	// ProviderID is the cloud provider id for the objects associated with the zone (the subnet on AWS)
 	ProviderID string `json:"id,omitempty"`
 
+	EgressID string `json:"egressId,omitempty"`
```

Comment:
Can we just call it `egress` at the json level (renaming the field also for sanity, but the JSON is what matters..)